src/components/WidthRestrictor/BlockWidthRestrictor/__tests__/BlockWidthRestrictor.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, RenderResult } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BlockWidthRestrictor } from '../BlockWidthRestrictor';
+
+describe('BlockWidthRestrictor', () => {
+    let view: RenderResult;
+
+    it('should render nothing if no children are provided', () => {
+        view = render(<BlockWidthRestrictor />);
+
+        expect(view.container).toBeEmptyDOMElement();
+    });
+
+    it('should render correctly', () => {
+        view = render(<BlockWidthRestrictor>Some children</BlockWidthRestrictor>);
+
+        expect(view.container).toMatchSnapshot();
+        expect(view.container).toHaveTextContent('Some children');
+        expect(view.container.children[0].tagName).toBe('DIV');
```

Comment:
I'm not sure about this approach. Could we use it ?

Reply:
First, I would write it as `expect(screen.findByRole('...').toBeInTheDocument`. But we should always ask ourselves: is this test useful? what concerns does it answer? etc.

Here you suggest to check if there is a `div` somewhere in the DOM. This doesn't tell us anything about what the user experiences/sees. So I don't see value in it. The test you wrote before, that checks if the text is present is a lot more valuable. I think that is enough for this check. this is also what you would do if you do a manual test: you would look for the text to be somewhere on the page. So I think we can indeed drop this line.

Reply:
:+1: